src/JavascriptVM/vm.ts

```diff
@@ -233,7 +233,12 @@ export class BlocklyInterpreter {
       );
     });
 
+    // this may cause an infinite loop, we should bail out if this is the case
+    this.interpreter.run();
+
     // Start running in a paused state - ie, spawn a "thread"
+    this.interpreter.appendCode("start();");
+
```

Comment:
I think it would be nice if the 'view source' component also shows that 'start()' gets called - currently
it looks like we just define a function, without running it.

Reply:
that might be ok, think of it like `main();`